features = ["mysql"] can't run

tokio = { "version" = "1.32.0", features = ["full"] }
sqlx-adapter = {version = "0.4.2",features = ["mysql"]}

```rust
   Compiling sqlx-adapter v0.4.2
error[E0428]: the name `ConnectionPool` is defined multiple times
  --> /Users/vincent/.cargo/registry/src/rsproxy.cn-8f6827c7555bfaf8/sqlx-adapter-0.4.2/src/actions.rs:22:1
   |
19 | pub type ConnectionPool = sqlx::PgPool;
   | --------------------------------------- previous definition of the type `ConnectionPool` here
...
22 | pub type ConnectionPool = sqlx::MySqlPool;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `ConnectionPool` redefined here
   |
   = note: `ConnectionPool` must be defined only once in the type namespace of this module

error[E0428]: the name `new` is defined multiple times
  --> /Users/vincent/.cargo/registry/src/rsproxy.cn-8f6827c7555bfaf8/sqlx-adapter-0.4.2/src/actions.rs:70:1
   |
28 | pub async fn new(conn: &ConnectionPool) -> Result<PgQueryResult> {
   | ---------------------------------------------------------------- previous definition of the value `new` here
...
70 | pub async fn new(conn: &ConnectionPool) -> Result<MySqlQueryResult> {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `new` redefined here
   |
   = note: `new` must be defined only once in the value namespace of this module

```

--- a/README.md
+++ b/README.md
@@ -26,8 +26,10 @@ ALTER TABLE casbin_rules RENAME TO casbin_rule;
 
 Add it to `Cargo.toml`
 
-```rust
-sqlx-adapter = { version = "0.4.2, features = ["postgres"] }
+```toml
+sqlx-adapter = { version = "0.4.2", default-features = false, features = ["postgres", "runtime-tokio-native-tls"]}
+# sqlx-adapter = { version = "0.4.2, default-features = false, features = ["mysql", "runtime-tokio-native-tls"]}
+# sqlx-adapter = { version = "0.4.2, default-features = false, features = ["sqlite", "runtime-tokio-native-tls"]}
 tokio = { version = "1.1.1", features = ["macros"] }
 ```
 

--- a/README.md
+++ b/README.md
@@ -10,8 +10,8 @@ Sqlx Adapter is the [Sqlx](https://github.com/launchbadge/sqlx) adapter for [Cas
 
 Based on [Sqlx](https://github.com/launchbadge/sqlx), The current supported databases are:
 
-- [Mysql](https://www.mysql.com/)
-- [Postgres](https://github.com/lib/pq)
+- [MySQL](https://www.mysql.com/)
+- [PostgreSQL](https://github.com/lib/pq)
 - [SQLite](https://www.sqlite.org)
 
 ## Notice
@@ -24,12 +24,26 @@ ALTER TABLE casbin_rules RENAME TO casbin_rule;
 
 ## Install
 
-Add it to `Cargo.toml`
+Add the following to `Cargo.toml`:
+
+For MySQL:
+
+```toml
+sqlx-adapter = { version = "0.4.2, default-features = false, features = ["mysql", "runtime-tokio-native-tls"]}
+tokio = { version = "1.1.1", features = ["macros"] }
+```
+
+For PostgreSQL:
 
 ```toml
 sqlx-adapter = { version = "0.4.2", default-features = false, features = ["postgres", "runtime-tokio-native-tls"]}
-# sqlx-adapter = { version = "0.4.2, default-features = false, features = ["mysql", "runtime-tokio-native-tls"]}
-# sqlx-adapter = { version = "0.4.2, default-features = false, features = ["sqlite", "runtime-tokio-native-tls"]}
+tokio = { version = "1.1.1", features = ["macros"] }
+```
+
+For SQLite:
+
+```toml
+sqlx-adapter = { version = "0.4.2, default-features = false, features = ["sqlite", "runtime-tokio-native-tls"]}
 tokio = { version = "1.1.1", features = ["macros"] }
 ```
 

